exercises/E-conditionals/exercise.js

```diff
@@ -9,6 +9,12 @@
 var name = "Daniel";
 var danielsRole = "mentor";
 
+if (danielsRole === "mentor") {
+  console.log("Hi, I'm Daniel, I'm a mentor.");
+} else {
```

Comment:
I works. my minor offer for best practice that "else if " can be preferred rather than "else" to code  'If Daniel is a student, ...' condition.